### ISSUE TITLE
Finding correct installation media 

### ISSUE THREAD

**Issue by martymotts** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#2652]
I am able to run the powershell script to create the installation media up to the point where it asks for Windows 10 Enterprise installation media. I have LTSB, which I am now aware does not work, but I cannot find Windows 10 Enterprise on the volume licensing service centre. 

Can Windows 10 IoT Enterprise SAC (ESD) be used on the skype system?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6f8b204f-9650-d268-8566-7a75ba0a1b6a
* Version Independent ID: 9a963e63-6716-7522-8abe-c8a407d3a46d
* Content: [Configure a Microsoft Teams Rooms console](https://docs.microsoft.com/en-us/MicrosoftTeams/room-systems/console#feedback)
* Content Source: [Teams/room-systems/console.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/room-systems/console.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by martymotts** [comment]
Error message when I try using Win10 Enterprise Education:

Invoke-Native : Command '& robocopy /mir 'C:\Users\mmotyka.FOODCOAU\Desktop\Windows 10' 'D:\' /R:0 /xf install.wim'
failed test '$_ -lt 8 -and $_ -ge 0' with code '11'.
At C:\Users\mmotyka.FOODCOAU\Desktop\CreateSrsMedia.ps1:560 char:4
+   (Invoke-Native "& robocopy /mir '$Src' '$Dst' /R:0 $Flags" $robocop ...
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-Native

**Comment by martymotts** [comment]
Invoke-Native : Command '& robocopy /mir 'C:\Users\xxxxxxxxxxxxxx\Desktop\Windows 10' 'D:\' /R:0 /xf install.wim'
failed test '$_ -lt 8 -and $_ -ge 0' with code '11'.
At C:\Users\xxxxxxxxxxxx\Desktop\CreateSrsMedia.ps1:560 char:4
+   (Invoke-Native "& robocopy /mir '$Src' '$Dst' /R:0 $Flags" $robocop ...
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Invoke-Native

**Comment by jambirk** [comment]
@travis-snoozy please comment

**Comment by jambirk** [comment]
@scanum please follow up with Travis on this issue

**Comment by tFable** [comment]
I'm having the exact same issue as martymotts. Trying to build the USB stick on Win 10 1903 machine.

**Comment by Travis-Snoozy** [comment]
LTSB, as mentioned originally, is not supported.

At time of writing, 1903 is not supported.

Non-OEMs must use Windows 10 Enterprise, matching the exact build version that CreateSrsMedia indicates when it runs.

**Comment by tFable** [comment]
Hi @Travis-Snoozy ,

Thanks for your reply.

I am using a supported Windows version to provide to the script (Win10 Ent. 1803). 1903 was mentioned because that's what the machine that the script is being executed on is running?

Does the Windows version of the machine that the script is being executed on match the Windows version that is being provided to the script to build the media? In other words, do I need to build the media from a win10 ent. 1803 machine?

**Comment by Travis-Snoozy** [comment]
Unfortunately It's Complicated(TM). It would be best to help eliminate potential version-over-version problems with DISM; I can't recall offhand if there's an issue with that particular pairing. If you're getting an error, please attach the PowerShell transcript (the script will indicate the file path of the transcript both at the start and end if its output) for analysis.

**Comment by tFable** [comment]
Thanks, @Travis-Snoozy . 

Please see the powershell error log attached as it was produced by the script.
I can try building using an 1803 machine tomorrow.

[PowerShell_transcript.txt](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/files/3484640/PowerShell_transcript.txt)

**Comment by Travis-Snoozy** [comment]
Robocopy is failing. Make sure that your destination media is large enough to fit all the files, and that none of the source files are larger than 4GiB individually. Try running the command that's showing as erroring out (`robocopy /mir 'c:\src\1803' 'D:\' /R:0 /xf install.wim`), and see what Robocopy says (I'm going off memory, but you may be able to get a similar result by running the script with `-Verbose` as well). The script is definitely saving you from getting a bad output stick/partial copy.

**Comment by tFable** [comment]
Hi, @Travis-Snoozy,

The destination flash drive has a capacity of 64GB's

I am not selecting any source files myself. I provided the script with the root of the mounted ISO for the Win10 Ent 1803 media (I've also tried extracting files to folder, with same results) and the script is deciding what to copy. 

If I'm using the correct Win10 media (the script is evaluating that) doesn't that also mean that the 4GiB limit that the script requires for the files is covered?

I'm going to try to run the script with the -verboe clause and reply back here with the results.

**Comment by tFable** [comment]
@Travis-Snoozy , attached is the results when running the script with "-verbose"

[PowerShell_transcript.txt](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/files/3484706/PowerShell_transcript.txt)

**Comment by Travis-Snoozy** [comment]
There nominally shouldn't ever be files bigger than 4GiB aside from (possibly) Install,wim, which is explicitly excluded for that reason.

That output is not including robocopy's messages, so you'll need to run the robocopy command by hand as mentioned above, and see where and why it's erroring out. If it's not something obvious like a too-big file, then the next-best guess is a bad USB drive or a flaky port (which are issues the script can't fix).

**Comment by Travis-Snoozy** [comment]
Ah, it's `-Debug` not `-Verbose`, what I get for going off of memory.

**Comment by tFable** [comment]
Hi @Travis-Snoozy ,

I believe the first log I posted was a -debug but here's a new one.

Thanks,
t

[PowerShell_transcript.txt](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/files/3487674/PowerShell_transcript.txt)

**Comment by tFable** [comment]
As @Travis-Snoozy suggested, I ran the robocopy command outside of the script. I was able to find the issue. It was Trend blocking the copy of autorun.inf:

`                           4    c:\src\1803\
            New File                 128        autorun.inf
2019/08/09 13:52:35 ERROR 5 (0x00000005) Copying File c:\src\1803\autorun.inf
Access is denied.`

I unloaded Trend from my PC and I was able to get past this step. Now the script is continuing through the rest of the process...

Thanks for your guidance!
